common/changes/@microsoft/api-extractor/emit-report-per-release-level_2024-04-02-22-39.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-extractor",
+      "comment": "Support creating API reports for each release level, in addition to a complete, \"untrimmed\" report",
```

```suggestion
      "comment": "Add support for \"variants\" of API reports which include or exclude items by release tag",
```

For a newcomer, it's not at all obvious what "variant" means in this context.  We should use the term "variant" consistently everywhere this feature is mentioned, and probably avoid the term "trimming" which refers to a fairly unrelated feature involving .d.ts rollups.